src/main/java/com/powsybl/openloadflow/ac/AcLoadFlowParameters.java

```diff
@@ -30,9 +30,7 @@ public class AcLoadFlowParameters extends AbstractLoadFlowParameters<AcLoadFlowP
 
     private AcEquationSystemCreationParameters equationSystemCreationParameters = new AcEquationSystemCreationParameters();
 
-    private NewtonRaphsonParameters newtonRaphsonParameters = new NewtonRaphsonParameters();
-
-    private NewtonKrylovParameters newtonKrylovParameters = new NewtonKrylovParameters();
+    private AcSolverParameters acSolverParameters = new NewtonRaphsonParameters();
```

Comment:
By default, we have NR parameters for Ac Solver Parameters?

Reply:
Indeed, would this be a problem ? Why would NR need NK parameters and vice-versa ?

Reply:
Yes It makes sense, I was making sure I understood the principle